Change the transcript to use TLS 1.3-style Handshake messages. Fixes (#262)
tlswg/dtls13-spec: draft-ietf-tls-dtls13.md

            case new_session_ticket:    NewSessionTicket;
            case key_update:            KeyUpdate;
        } body;
    } DTLSHandshake;
~~~

In DTLS 1.3, the message transcript is computed over the original
TLS 1.3-style Handshake messages without the message_seq,
fragment_offset, and fragment_length values. Note that this is

> a change from DTLS 1.2 where which those values were included

Reviewer: typo
```suggestion
a change from DTLS 1.2 where those values were included
```

When transmitting the handshake message, the sender divides the
message into a series of N contiguous data ranges. The ranges MUST NOT
overlap.  The sender then creates N DTLSHandshake messages, all with the

> same message_seq value asssociated the original handshake message.  Each new

Reviewer 2: Typo and I think missing a word. Did you mean this?
```suggestion
same message_seq value associated with the original handshake message.  Each new
```

When a DTLS implementation receives a handshake message fragment corresponding
to the next expected handshake message sequence number, it
MUST process it, either by buffering it until it has the entire handshake message

> or by processing any in-order portions of the message

Reviewer 3: ```suggestion
or by processing any in-order portions of the message.
```

message should be treated through an independent state machine after inspecting the handshake
message type.

##  CertificateVerify and Finished Messages

CertificateVerify and Finished messages have the same format as in

> TLS 1.3.  Hash calculations include entire handshake messages, including

Reviewer 4: I mostly assume no one implemented the literal reading of this, where CertificateVerify/Finished do not have the fragment-specific fields

Author: Well that's what we are telling people to do now.
